.github/workflows/node.js.yml

```diff
@@ -1,37 +1,75 @@
-name: CI
+name: Node
 
 on:
   pull_request:
   push:
     branches-ignore: "dependabot/**"
 
+env:
+  NODE_VERSION: 20.x
+
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
+    name: ${{ matrix.MIGRATE_VERSION }} - jQuery ${{ matrix.JQUERY_VERSION }} (Chrome/Firefox)
+    strategy:
+      fail-fast: false
+      matrix:
+        MIGRATE_VERSION: ["min"]
+        JQUERY_VERSION: ["3.x-git.min", "3.x-git.slim.min"]
```

Comment:
Can we test against `git.min` & `git.slim.min` as well? It shouldn't be too long and it'd be great to catch potential issues directly at the PR level.

If you change it here then remember also about the Safari entry.